contracts/src/v0.8/KeeperRegistry1_2.sol

```diff
@@ -11,16 +11,16 @@ import "./interfaces/TypeAndVersionInterface.sol";
 import "./interfaces/AggregatorV3Interface.sol";
 import "./interfaces/LinkTokenInterface.sol";
 import "./interfaces/KeeperCompatibleInterface.sol";
-import "./interfaces/KeeperRegistryInterface.sol";
```

Comment:
I think I still need to update the interface names in these files? I only updated the file names but in the future we will add new interfaces here so there may be naming conflicts?

Reply:
I think we only need one of two things
1. Either have different interface names and struct names within the same file
2. Or have different files (your current implementation) with same struct names

I think 2 is more cleaner and is what you already have